assets/javascripts/discourse/widgets/discourse-reactions-actions.js.es6

```diff
@@ -122,21 +121,18 @@ export default createWidget("discourse-reactions-actions", {
       classes.push("has-reacted");
     }
 
-    if (attrs.post.current_user_used_main_reaction) {
+    if (post.current_user_used_main_reaction) {
       classes.push("has-used-main-reaction");
     }
 
     if (
-      attrs.post.likeAction &&
-      (attrs.post.likeAction.canToggle || attrs.post.likeAction.can_undo)
+      post.likeAction &&
+      (post.likeAction.canToggle || post.likeAction.can_undo)
     ) {
       classes.push("can-toggle-main-reaction");
     }
 
-    if (
-      attrs.post.reactions.length === 1 &&
-      attrs.post.reactions[0].id === mainReaction
-    ) {
+    if (post.reactions.length == 1 && post.reactions[0].id === mainReaction) {
```

Comment:
```suggestion
    if (post.reactions.length === 1 && post.reactions[0].id === mainReaction) {
```

Reply:
Hi Joffrey, I directly committed this change from here, as my laptop is not working from this evening. I am setting up my brother's spare pc right now and will give my laptop for repairing tomorrow morning. Hope you wont mind if there's some delay in changes😅